examples/qml_minimal/CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ if(BUILD_TESTING)
             NAME ${APP_TEST_NAME}
             COMMAND ${TEST_CMD}
         )
+        set_property(TEST ${APP_TEST_NAME} PROPERTY ENVIRONMENT "QT_QPA_PLATFORM=offscreen")
 
```

Comment:
you could also add it to the qt_types_standalone test incase that starts testing widgets / qml objects enough that they try to show anything. And that could go in the main.cpp function i guess

Reply:
Done.